crates/kubelet/src/pod/status.rs

```diff
@@ -213,6 +239,12 @@ pub enum Phase {
     Unknown,
 }
 
+impl std::fmt::Display for Phase {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", serde_json::json!(self).as_str().unwrap())
```

Comment:
I'm not sure what implementation you are referring to. I don't see that `PodStatus` implements `Display`, and phase is simply stored as a `String`. The reason I did it this way is that as far as I know this is the only way to automatically generate `Display` for `enums`, rather than having to specify the exact string used. You can also use `Debug`, but `serde` lets us control the capitalization, etc. a little more. I don't think this can panic. I can change this to use `Debug` but I don't like the idea of having to hard code the strings and update them if the `enum` changes.

Reply:
Oh wait, I somehow misread `Phase` as `Status` when I was going through your code 🤦. Sorry